cumulusci/tasks/dx_convert_from.py

```diff
@@ -0,0 +1,27 @@
+import shutil
+from pathlib import Path
+
+from cumulusci.tasks.sfdx import SFDXBaseTask
+
+
+class DxConvertFrom(SFDXBaseTask):
+    task_options = {
+        "extra": {"description": "Append additional options to the command"},
+        "src_dir": {
+            "description": "The path to the src directory where converted contents will be stored. Defaults to src/"
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+        if "src_dir" not in self.options:
+            self.options["src_dir"] = "src"
```

Comment:
I would put this default in cumulusci.yml instead of Python so that it shows up in `cci task info dx_convert_from`